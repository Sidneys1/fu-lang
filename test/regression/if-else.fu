main: u8() = {
  if (false)
    return 1;

  if (false)
    return 2;
  else if (false)
    return 3;
  else if (false)
    return 4;
  else
    return 5;
};
