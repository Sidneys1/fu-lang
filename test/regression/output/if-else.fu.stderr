INFO:fu.compiler:Output binary `a.out`: 878 Bytes
